specification/machinelearningservices/AzureAI.Evaluation/model.tsp

```diff
@@ -133,6 +134,22 @@ model UpdateEvaluationRequest {
   description: string;
 }
 
+@doc("Extended Evaluation with additional fields.")
+model EvaluationSchedule extends Evaluation {
+  @doc("Recurrence pattern for the evaluation.")
+  recurrence: Recurrence;
+
+  @doc("Sampling strategy for the evaluation.")
+  samplingStrategy: SamplingStrategy;
+}
+
+model EvaluationScheduleResult {

```

Comment:
Can we match this model used for EvaluationSchedule ?

Reply:
removed EvaluationScheduleResult and moved to different api's for listing schedules and jobs under a schedule